tests/Avalonia.RenderTests/TestBase.cs

```diff
@@ -81,13 +81,17 @@ protected void CompareImages([CallerMemberName] string testName = "")
         {
             string expectedPath = Path.Combine(OutputPath, testName + ".expected.png");
             string actualPath = Path.Combine(OutputPath, testName + ".out.png");
-            MagickImage expected = new MagickImage(expectedPath);
-            MagickImage actual = new MagickImage(actualPath);
-            double error = expected.Compare(actual, ErrorMetric.RootMeanSquared);
-
-            if (error > 0.022)
+            using (MagickImage expected = new MagickImage(expectedPath))
             {
-                Assert.True(false, actualPath + ": Error = " + error);
+                using (MagickImage actual = new MagickImage(actualPath))
```

Comment:
Rather than nesting the `using` statements like this, could you use the standard multiple-`using` pattern?

http://stackoverflow.com/a/1329765/6448